FreeRTOS-Plus/Test/CBMC/proofs/ARP/ulARPRemoveCacheEntryByMAC/Makefile.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ENTRY": "ulARPRemoveCacheEntryByMac",
+  "NUMBER_OF_ENTRIES":1,
+  "CACHE_ENTRIES":6,
+  "CBMCFLAGS":
+  [
+  ],
+  "OBJS":
+  [
+    "$(ENTRY)_harness.goto",
+    "$(FREERTOS)/../FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.goto"
```

Comment:
Did we change FREERTOS to point to the root, so this should be FREERTOS/FreeRTOS-Plus?

Reply:
I would like to change the root in another PR. These PRs are meant to add proofs and I believe that they should just do that.
- Learning to keep my PRs single purpose :)

Reply:
ok